### PR TITLE
Add MAILER_URL env var to Heroku config file

### LINKED ISSUE #686: failed to deploy on heroku
`ATTENTION: This operation should not be executed in a production environment.

Creating database schema...
Database schema created successfully!

[37;41m                                                                          [39;49m
[37;41m  [Symfony\Component\DependencyInjection\Exception\EnvNotFoundException]  [39;49m
[37;41m  Environment variable not found: "MAILER_URL".                           [39;49m
[37;41m                                                                          [39;49m

[32mdoctrine:schema:create [--dump-sql] [--em [EM]] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] [39m[32m<command>[39m`

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #686.

### CODE REVIEW

**Comment by yceruto** [comment]
[https://github.com/symfony/demo/issues/686#issuecomment-340009938] Also we might need to add `DATABASE_URL` to env section.

File: app.json [hunk 1]
```diff
@@ -22,11 +22,19 @@
...
             "value": "0"
         },
-        "SYMFONY_LOG": "php://stderr",
         "APP_SECRET": {
             "description": "Extra entropy for %kernel.secret%; used for CSRF tokens, cookies and signed URLs.",
             "generator": "secret"
-        }
+        },
+        "DATABASE_URL": {
+            "value": "sqlite:///var/data/blog.sqlite",
```
[the remark this answers is not in the document]
Comment (stof) [reply app.json]: using SQLite on Heroku won't work fine:

- it is incompatible with dyno scaling (as each dyno gets its own DB)
- it is incompatible with dyno cycling (Heroku replaces the dyno on a daily basis, and the new one would gets its own empty DB).

You should remove this `DATABASE_URL` env variable. The app.json is adding the `heroku-postgresql` addon, which adds such an env variable already.

[the remark this answers is not in the document]
Comment (javiereguiluz) [reply app.json]: I don't use Heroku ... but there may be something wrong about DATABASE_URL. See #686.

[the remark this answers is not in the document]
Comment (stof) [reply app.json]: well, the question is, do they have the addon or no ?

**Pull request closed without merging**